cypress/e2e/homepage.feature

```diff
@@ -2,4 +2,5 @@ Feature: homepage
 
 Scenario: Create new project should be a success
   Given I visit the "/"
-  Then I expect current url is "/"
```

Comment:
1 commit for all e2e test modification could be better. Even better, put all of them in the [Setup HomePage](https://github.com/ditrit/leto-modelizer/pull/25/commits/8fe3ecdd9073a9b21e3980bbda90423c713a284a) commit, like unit tests.